src/DDTrace/Integrations/ZendFramework/ZendFrameworkIntegration.php

```diff
@@ -65,7 +65,16 @@ public static function load()
             $args = func_get_args();
             $options = $args[0];
 
```

Comment:
@labbati @SammyK I think one of you guys have a bit more experience with ZendFramework's autoloader.

I wonder what would be the best way to load these files - I've tried a few approaches, and each fails some existsting test.

Before I dig deeper, I wonder if you could share some insights

Reply:
Hum... is there a specific error that's happening when you manually include? I can't think of anything off the top of my head that would problematic from the digging we did with the ZF1 autoloader a few months ago.

Reply:
seems like just manually including was the reason. I didn't think that the file I wanted to include was not loaded by our old autoloader.

While working on a solution I decided to implement two Autoloaders.
One which loads all required things then removes itself, the second one is based on how composer "compiles" autoloaders.

I think it still should give performance improvement, while still allows some files not to be loaded